main.go

```diff
@@ -1,5 +1,5 @@
 //go:generate go run pkg/codegen/cleanup/main.go
-//go:generate go run pkg/codegen/main.go
+//go:generate go run pkg/codegen/main.go pkg/codegen/template.go
```

Comment:
Is this change needed? that file is part of the main package and doesn't run any code

Reply:
Yes, it is required: `go run` doesn't look at files not passed as arguments when compiling. Without this change, the template isn't found because it is in `pkg/codegen/template.go`.